static/app/utils/fields/index.ts

```diff
@@ -827,8 +827,8 @@ export const ALLOWED_EXPLORE_VISUALIZE_FIELDS: SpanIndexedField[] = [
 ];
 
 export const ALLOWED_EXPLORE_VISUALIZE_AGGREGATES: AggregationKey[] = [
-  AggregationKey.AVG,
   AggregationKey.COUNT,
```

Comment:
Can we just add a comment here that we implicitly use the first element in this list as the default aggregate field? And that we're working under the assumption that it's count and swapping the arg based on it.